Networking/Networking/Model/StoreOnboardingTask.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+public struct StoreOnboardingTask: Decodable, Equatable {
+    public let isComplete: Bool
+    public let type: TaskType
+
+    private enum CodingKeys: String, CodingKey {
+        case isComplete
+        case type = "id"
+    }
+}
+
+public extension StoreOnboardingTask {
+    enum TaskType: Decodable, Equatable {
+        case addFirstProduct
+        case launchStore
+        case customizeDomains
+        case payments
+        case unsupported
```

Comment:
non-blocking: do you think we can keep the value of the task for easier debugging/development? similar to how [`ProductType` can take in a custom type](https://github.com/woocommerce/woocommerce-ios/blob/faebeb54d4c06f40b730608ab3ebe65e2023c50b/Networking/Networking/Model/Product/ProductType.swift#L18)

Reply:
Good idea. Stored the `id` as string in https://github.com/woocommerce/woocommerce-ios/pull/9017/commits/9e0152583a84f8f191ca4db66434f548f99443f3